implementation/src/main/java/io/smallrye/health/HealthStatus.java

```diff
@@ -0,0 +1,67 @@
+package io.smallrye.health;
+
+import java.util.function.BooleanSupplier;
+import org.eclipse.microprofile.health.HealthCheck;
+import org.eclipse.microprofile.health.HealthCheckResponse;
+
+public class HealthStatus {
+
+    protected static final String DEFAULT_HEALTH_CHECK_NAME = "health-check";
```

Comment:
Use something ugly, like a random UUID? Or at least `anonymous-health-check` or `unnamed-health-check`?

Reply:
What I see is that a UUID means some time spent on calculating it. Not much but I am thinking in terms of Quarkus, as it is a static method it will be calculated on boot time which means more boot up time. So we can either go to `anonymous-health-check` or we do some kind of lazy init when these methods are called. WDYT?

Reply:
the UUID will need to be recalculated on each /health/* invocation to get distinct ids :)

Reply:
Ok I get your point now, yeah for example one for readiness and another one for liveness. Although I think it should be perfectly valid to have the same name in both cases, I agree that setting a random as default is better. I am going to update the PR with UUID calculation in case of not setting the name.

Reply:
@xstefank @Ladicek I have updated the PR with a random name generator.